recipes-core/josev/python3-iso15118_2023.7.0.bb

```diff
@@ -16,6 +16,14 @@ RDEPENDS:${PN} += " \
     ${PYTHON_PN}-pydantic \
     ${PYTHON_PN}-psutil \
     ${PYTHON_PN}-py4j \
+    ${PYTHON_PN}-aiofile \
+    ${PYTHON_PN}-environs \
+    ${PYTHON_PN}-marshmallow \
+    ${PYTHON_PN}-netifaces \
+    ${PYTHON_PN}-python-dotenv \
+    ${PYTHON_PN}-typing-extensions \
+    ${PYTHON_PN}-caio \
+    ${PYTHON_PN}-packaging \
 "
 
```

Comment:
in order to be able to join nto the system would be nice to have `debug-tweaks`

Reply:
isn't that typically something you specify in your local.conf?
Simply adding
```bitbake
EXTRA_IMAGE_FEATURES += "debug-tweaks"
```
There should be enough

Reply:
👍 yup understood : )